src/assets/css/global/all.css

```diff
@@ -13,10 +13,8 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
-@media only screen and (max-width: 600px) {
-  html {
-    font-size: 14px;
-  }
+html {
```

Comment:
Changed this to better align with the tailwind media queries. Otherwise, I would have needed to do custom media queries at 600px to make things like alignment for the new tree structure to work.